# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
# goal definition
sensor_msgs/JointState goal_state
duration pose_duration
duration move_duration
